lnrpc/walletrpc/walletkit.proto

```diff
@@ -1230,6 +1229,11 @@ message BumpFeeRequest {
     retained.
     */
     uint64 budget = 7;
+
+    // Optional. The deadline delta in number of blocks that the output
+    // should be spent within. This translates internally to the width of the
+    // fee function that the sweeper will use to bump the fee rate.
```

Comment:
We may add one sentence in the end saying `When the deadline is reached, ALL the budget will be spent as fees`.